### ISSUE TITLE
Comments in Dictionary entries causes a CommentPersistenceViolation

### ISSUE THREAD

**Issue by ScottLongley** [Scony/godot-gdscript-toolkit#112]
gdformat version: 3.2.7, 3.2.8, master (at time of writing)

```GDScript
func _ready():
	var tmp = {
		"key": "value",
		# this comment will trip CommentPersistenceViolation
	}
```

Hope this helps, thanks for all of your hard work!

**Comment by Scony** [comment]
this is not implemented basically.

**Comment by Scony** [comment]
it will be addressed in #30
